src/main/java/com/project/sistemaStock/controller/UserController.java

```diff
@@ -9,9 +9,7 @@
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
 
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Optional;
+import java.util.*;
```

Comment:
esto no es buenas practicas!!! be carefull

Reply:
entre el map, el hashmap, el uui, etc, son varios imports de utils, mejor el *